sdk/textanalytics/Azure.AI.TextAnalytics/tests/AnalyzeOperationTests.cs

```diff
@@ -0,0 +1,410 @@
+            {
+                Assert.IsTrue(entitiesList.Contains(entity.Text));
+                Assert.IsNotNull(entity.Category);
+                Assert.IsNotNull(entity.Offset);
+                Assert.IsNotNull(entity.ConfidenceScore);
+            }
+
+            // PII
+
+            Assert.AreEqual(2, entitiesResult.Count);
+
+            Assert.AreEqual(3, entitiesResult[0].Entities.Count);
+            Assert.IsNotNull(entitiesResult[0].Id);
+            Assert.IsNotNull(entitiesResult[0].Entities);
+            Assert.IsNotNull(entitiesResult[0].Error);
+
+        }
+
+        [Test]
+        public async Task AnalyzeOperationWithSkipParameter()
+        {
+            TextAnalyticsClient client = GetClient();
+
+            AnalyzeOperationOptions operationOptions = new AnalyzeOperationOptions()
+            {
+                KeyPhrasesTaskParameters = new KeyPhrasesTaskParameters()
+                {
+                    ModelVersion = "latest"
+                },
+                DisplayName = "AnalyzeOperationWithSkipParameter",
+                Skip = 1
+            };
+
+            AnalyzeOperation operation = await client.StartAnalyzeOperationBatchAsync(batchConvenienceDocuments, "en", operationOptions);
+
+            await operation.WaitForCompletionAsync();
+
+            AnalyzeOperationResult resultCollection = operation.Value;
+
+            ExtractKeyPhrasesResultCollection result = resultCollection.KeyPhraseResult;
+
+            Assert.IsNotNull(result);
+
+            // TODO - Update this to Assert.AreEqual(1, result.Count), once skip starts working.
+            Assert.AreEqual(2, result.Count);
+
+            var keyPhrasesListId2 = new List<string> { "Tesla stock", "year" };
+
+            // TODO - Update this to result[0].KeyPhrases
+            foreach (string keyphrase in result[1].KeyPhrases)
+            {
+                Assert.IsTrue(keyPhrasesListId2.Contains(keyphrase));
+            }
+        }
+
+        [Test]
+        public async Task AnalyzeOperationWithTopParameter()
+        {
+            TextAnalyticsClient client = GetClient();
+
+            AnalyzeOperationOptions operationOptions = new AnalyzeOperationOptions()
+            {
+                KeyPhrasesTaskParameters = new KeyPhrasesTaskParameters()
+                {
+                    ModelVersion = "latest"
+                },
+                DisplayName = "AnalyzeOperationWithSkipParameter",
+                Top = 1
+            };
+
+            AnalyzeOperation operation = await client.StartAnalyzeOperationBatchAsync(batchConvenienceDocuments, "en", operationOptions);
+
+            await operation.WaitForCompletionAsync();
+
+            AnalyzeOperationResult resultCollection = operation.Value;
+
+            ExtractKeyPhrasesResultCollection result = resultCollection.KeyPhraseResult;
+
+            Assert.IsNotNull(result);
+
+            // TODO - Update this to Assert.AreEqual(1, result.Count), once skip starts working.
```

Comment:
is top not working either? I suggest also ignoring the whole test and write it as you will expect it to work